src/Avalonia.Controls/Platform/IScreenImpl.cs

```diff
@@ -1,11 +1,19 @@
 ﻿using System.Collections.Generic;
 
+#nullable enable
+
 namespace Avalonia.Platform
 {
     public interface IScreenImpl
     {
         int ScreenCount { get; }
 
         IReadOnlyList<Screen> AllScreens { get; }
+
+        Screen? ScreenFromWindow(IWindowBaseImpl window);
```

Comment:
Which Screen should we return if window intersects multiple ones?

Reply:
Isn't it something for OS to decide?